### ISSUE TITLE
registrationInfo is undefined Error

### ISSUE THREAD

**Issue by Djfe** [mozilla-services/screenshots#3982]
![2018-01-13_14-13-22](https://user-images.githubusercontent.com/5685512/35111703-21a77ef6-fc7c-11e7-910b-68e0c9c93b4e.png)

This happened on Januar 13th with the nightly from 13th or 12th or 11th. (sadly didn't write down the version number and forgot to report instantly)

EDIT: I just pressed the screenshot button and got the tutorial with this error on the bottomright.

Maybe it happened because of my mouse. (sometimes it doubleclicks, I still have to investigate that)

**Comment by johngruen** [comment]
Keep us posted if you are able to reproduce the error.

**Comment by johngruen** [comment]
@SoftVision-CosminMuntean will also try to repro

**Comment by Djfe** [comment]
I might not be able to since it happened randomly.
But if I find anything, then you'll get the info :)

**Comment by SoftVision-CosminMuntean** [comment]
Unfortunately I haven't managed to reproduce this issue. The only errors I got are the "catcher is not defined" and "Attempted to call slides.display() twice", which are already tracked.
For testing this I have used a new clean Firefox profile where the Tour was never visited and I tried to activate it using multiple scenarios (from context menu, Page Actions menu, with double clicks, etc...).
However I will continue to investigate this.
